### DISCUSSION TITLE
Support INP for reportWebVitals()

### DISCUSSION THREAD

**Discussion by fmacherey** [vercel/next.js#38407] [Ideas]
### Describe the feature you'd like to request

The `reportWebVitals` function should support the new INP metric from web-vitals package. 

For now, this is only supported by web-vitals@next (v3) https://github.com/GoogleChrome/web-vitals/tree/next 
But as this metric is more and more important, next should support this. 

### Describe the solution you'd like

s.o.

### Describe alternatives you've considered

For now, we use the web-vitals package directly, which is kind of redundant to the internal next web-vitals logging.

**Comment by balazsorban44** [comment]
This is being worked on https://github.com/vercel/next.js/pull/36490 :+1:

**Reply by housseindjirdeh** [reply]
It's been merged into Next v12.2.0. @fmacherey you should be able to use it now but let us know if you run into any issues!

**Reply by fmacherey** [reply]
@housseindjirdeh yes it works. Thanks for your quick replies. 

Do you have already an issue to update the docs? They don't mention INP yet.

**Reply by housseindjirdeh** [reply]
Probably should have opened an issue first, but https://github.com/vercel/next.js/pull/38464 should take care of it :)
